ui/components/app/assets/token-cell/token-cell.tsx

```diff
@@ -32,11 +34,12 @@ export default function TokenCell({
   const tokenImage = tokenData?.iconUrl || image;
   const formattedFiat = useTokenFiatAmount(address, string, symbol, {}, false);
   const locale = useSelector(getIntlLocale);
-  const primary = new Intl.NumberFormat(locale, {
-    minimumSignificantDigits: 1,
-    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-    // @ts-ignore
-  }).format(string.toString());
+  const primary =
+    new Intl.NumberFormat(locale, {
+      minimumSignificantDigits: 1,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+    }).format(string.toString()) || '0';
```

Comment:
Is this `|| '0'` needed for this change?

Reply:
i think so yes, the `format(string.toString()) || '0'` ensures that string is properly formatted and falls back to '0' if the value is `null` or `undefined`.